### DISCUSSION TITLE
How do I list my current theme?

### DISCUSSION THREAD

**Discussion by awa5114** [JanDeDobbeleer/oh-my-posh#1329] [Troubleshoot] [answered]
I set a theme for my powershell command prompt but forgot which theme this was. Is there a command I can use to list the current theme?

**Comment by JanDeDobbeleer** [comment, marked as answer]
You can have a look at your `$PROFILE`

**Reply by awa5114** [reply]
Thanks.

**Reply by lnu** [reply]
`get-poshcontext` should also do the trick

**Reply by lnu** [reply]
`export | grep POSH` on linux
